mentor_classifier/tests/test_ner.py

```diff
@@ -0,0 +1,59 @@
+#
+# This software is Copyright ©️ 2020 The University of Southern California. All Rights Reserved.
+# Permission to use, copy, modify, and distribute this software and its documentation for educational, research and non-profit purposes, without fee, and without a written agreement is hereby granted, provided that the above copyright notice and subject to the full license file found in the root of this software deliverable. Permission to make commercial use of this software may be obtained by contacting:  USC Stevens Center for Innovation University of Southern California 1150 S. Olive Street, Suite 2300, Los Angeles, CA 90115, USA Email: accounting@stevens.usc.edu
+#
+# The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+#
+from os import path
+
+import pytest
+import responses
+
+from .helpers import (
+    fixture_mentor_data,
+    load_mentor_csv,
+)
+from mentor_classifier.ner import NamedEntities
+from .helpers import get_answers
+from typing import List, Dict
+
+
+@pytest.fixture(scope="module")
+def shared_root() -> str:
```

Comment:
here's the `shared_root` fixture. Use it explicitly in tests where you need it and/or if you need the actual env var set, you can set it here with something like

```python
@pytest.fixture(scope="module")
def shared_root(monkeypatch) -> str:
  root = path.abspath(path.join("..", "shared", "installed"))
  # not sure the exact env-var name, but you get the idea
  monkeypatch.setenv("SHARED_ROOT", root)
  return root
  
```